underlay/js/bigipreport.js

```diff
@@ -2505,40 +2505,40 @@ function showDeviceOverview() {
                 else {
                     pollingStatus = 'N/A (passive device)';
                 }
-            }
-            if (firstDevice) {
+                if (firstDevice) {
+                    html +=
+                        `<tr><td rowspan="${deviceGroup.ips.length}" class="deviceiconcell">
+              <img class="deviceicon" alt="deviceicon" src="${icon}"/></td>` +
+                            `<td class="devicenamecell" rowspan="${deviceGroup.ips.length}">${deviceGroup.name}</td>`;
+                    firstDevice = false;
+                }
+                else {
+                    html += '<tr>';
+                }
+                const devicestatus = loadbalancer.color || 'red';
                 html +=
-                    `<tr><td rowspan="${deviceGroup.ips.length}" class="deviceiconcell">
```

Comment:
I think template strings with string interpolation is much more readable (ie. `My name is ${name}`).
Would it make sense to use that instead of concatenation?

Reply:
Fine with me. All I did was move the closing bracket so that the loadbalancer stuff is not rendered if loadbalancer is null.

Reply:
Then it's out of scope. I will approve and let you decide if you want to change or not.

Reply:
Then it's out of scope. I'll approve and let you decide if you want to change or not.